sdk/qumulo/Azure.ResourceManager.Qumulo/api/Azure.ResourceManager.Qumulo.net8.0.cs

```diff
@@ -124,25 +116,39 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
     }
     public partial class MarketplaceDetails : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Qumulo.Models.MarketplaceDetails>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Qumulo.Models.MarketplaceDetails>
     {
-        public MarketplaceDetails(string planId, string offerId, string publisherId) { }
+        public MarketplaceDetails(string planId, string offerId) { }
```

Comment:
So `publisherId` is no longer required in the new api-version.
This is not breaking from the spec's perspective, but we must do something on the .net code to make it non-breaking.

Please add a partial class with the same namespace and same name as this class, and put it in `sdk/qumulo/Azure.ResourceManager.Qumulo/src/Customized/Models`, and we need to add the missing members back, like this:
```
namespace Azure.ResourceManager.Qumulo
{
	// we will need the xml doc back, you should be able to find it in git history.
	public partial class MarketplaceDetails(string planId, string offerId, string publisherId) : this(planId, offerId)
	{
		PublisherId = publisherId;
	}
}
```